uobtheatre/venues/test/test_schema.py

```diff
@@ -22,6 +22,16 @@ def test_venues_schema(gql_client, gql_id):
               node {
                 id
                 name
+                address {
+                  id
+                  buildingName
+                  buildingNumber
+                  street
+                  city
+                  postcode
+                  latitude
+                  longitude
```

Comment:
Sorry @streth11 I have misled you. Keep just the id in here to make sure that the address is accessible. Then in addresses/test/test_schema add all the fields in an AddressNode. You will need to access the address through a venue in that new test but that is fine.

Reply:
i think i have fixed